tensorflow/python/ops/math_grad.py

```diff
@@ -767,6 +767,10 @@ def _SparseMatMul(t1, t2, out_dtype,
 def _FloorGrad(_, unused_grad):
   return [None]
 
+@ops.RegisterGradient("Rint")
```

Comment:
I think this probably deserves some comment (same with Floor).  Why is the gradient of "Rint" [None] ?  Is this one of those "It's not correct but it works" type of things, or should we even be registering a gradient here at all?